core/vm/contracts.go

```diff
@@ -124,6 +134,12 @@ func init() {
 	for k := range PrecompiledContractsBerlin {
 		PrecompiledAddressesBerlin = append(PrecompiledAddressesBerlin, k)
 	}
+	for k := range PrecompiledContractsCel2 {
+		PrecompiledAddressesCel2 = append(PrecompiledAddressesBerlin, k)
+	}
+	for k := range PrecompiledCeloContractsCel2 {
+		PrecompiledAddressesCel2 = append(PrecompiledAddressesBerlin, k)
```

Comment:
```suggestion
		PrecompiledAddressesCel2 = append(PrecompiledAddressesCel2, k)
```

Reply:
I moved these fixes into their original commit, since I would like to rebase+merge this PR into celo1, rather than squash it.